pkg/routeagent/controllers/route/route.go

```diff
@@ -419,36 +422,47 @@ func (r *Controller) processNextEndpoint() bool {
 			return fmt.Errorf("error retrieving submariner endpoint object %s: %v", name, err)
 		}
 
+		klog.V(log.TRACE).Infof("Processing endpoint %s : %s", name, endpoint.String())
```

Comment:
Don't specifically call `String()` so we don't incur the string conversion overhead on the fast path, ie when TRACE level is not enabled.

Reply:
True, :+1:

Reply:
Done